src/plugins/typechecker/README.md

```diff
@@ -0,0 +1,190 @@
+- infos = Information about the typechecker plugin is in keys below
+- infos/author = Armin Wurzinger <e1528532@libelektra.org>
+- infos/licence = BSD
+- infos/needs = 
+- infos/provides = typechecker
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained experimental global
+- infos/metadata =
+- infos/description = a plugin which typechecks configuration specifications
+
+## Introduction
+
+A plugin which typechecks specifications before setting keys and after getting keys from
+a mounted specification.
+
+The typesystem is based on regular expressions. Each key is assigned with a regex
```

Comment:
"uses regular expressions as foundation"?

"each key" or "each type"?

And why not write "The type checker assigns". (active over passive)